instance/static/html/instance/index.html

```diff
@@ -24,6 +24,13 @@
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
         <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Provisioning in progress" ng-switch-default></i>
+        </div>
```

Comment:
@chromano Doesn't that make it a switch with a single default element, ie always showing? I would have expected to only use the `instance.progress` attribute, and show "Provisioning in progress" when `instance.progress == instance.PROGRESS_RUNNING`? I feel like I'm missing something here.